extension/src/experiments/model/util.ts

```diff
@@ -8,8 +8,15 @@ type Value = undefined | null | [] | string | number
 const isDate = (value: Value): boolean =>
   !!(typeof value === 'string' && Date.parse(value))
 
-const isLongNumber = (value: Value): boolean =>
-  typeof value === 'number' && value.toString().length > 7
+export const formatNumber = (value: number): string => {
+  const defaultPrecision = 5 // for when we can't calculate real precision yet
```

Comment:
Don't we want to be consistent here and do length > 10 precision 8 for everything?

Reply:
> Don't we want to be consistent here and do length > 10 precision 8 for everything?

Oh ok! Apologies, I must have misunderstood yesterday's discussion 🤦‍♀️